### ISSUE TITLE
Есть ли статья по темам?

### ISSUE THREAD

**Issue by cyberS7** [bem/bem-forum-content-ru#957]
Добрый день. Подскажите, пожалуйста, есть ли статья по темам? Не хочу использовать  "islands". Как создавать свои темы?

**Comment by cyberS7** [comment]
Скопировал папку design, переименовал в "mydesign", внутри переименовал ctx = '.input_theme_mydesign' и так далее.
Проблема в том, что если я меняю стили скажем у инпута внутри темы, то не происходит пересборки, пока не выполню enb make - enb server. Это как-то печально.

**Comment by tadatuta** [comment]
@cyberS7 я правильно понимаю, что за основу взят `project-stub`, а папка `mydesign` осталась в папке `bem-components` внутри `libs`, при этом в конфиге сборки осталось, как было здесь https://github.com/bem/project-stub/blob/bem-core/.enb/make.js#L28-L29 `check: false`?

Если так, то нужно просто убрать `check: false`, чтобы сборщик стал следить за изменениями в этой папке. Ну и, конечно, есть смысл `mydesign` переложить к себе в проект из `bem-components` (+ соответственно исправить пути в конфиге сборки).

**Comment by cyberS7** [comment]
@tadatuta да, действительно, проблема была в этом: `check: false` Благодарю.
